src/usdb_syncer/gui/__init__.py

```diff
@@ -73,6 +80,38 @@ def _run() -> None:
     app.exec()
 
 
+def get_latest_version() -> str | None:
```

Comment:
I would move these functions outside the gui module.

Reply:
To utils?

Reply:
I moved both to utils.